src/Input/ToggleSwitch/ToggleSwitch.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+
+import { Icon, SecondaryColor } from '../../index';
+import { Toggle, ToggleBall, ToggleSwitchWrapper, ToggleSwitchProps } from './';
+
+const ToggleSwitch: React.FunctionComponent<ToggleSwitchProps> = ({
+  defaultActive,
+  activeIconName,
+  inactiveIconName,
+  activeBackgroundColor,
+  inactiveBackgroundColor,
+  activeBallColor,
+  inactiveBallColor,
+  onChange,
+}) => {
+  const [active, setActive] = React.useState(defaultActive || false);
+
+  return (
+    <ToggleSwitchWrapper>
+      <Toggle
+        onClick={() => {
+          const newActiveState = !active;
+          setActive(newActiveState);
+          if (onChange) onChange(newActiveState);
+        }}
+        active={active}
+        activeBackgroundColor={activeBackgroundColor}
+        inactiveBackgroundColor={inactiveBackgroundColor}
+      >
```

Comment:
Accessibility is a key feature for component libraries. I'm not super familiar with it as well, but what I usually do is to copy the `aria` attributes from similar components. I suggest copying the them by inspecting this component found here https://scottaohara.github.io/aria-switch-control/

![image](https://user-images.githubusercontent.com/35050056/68006754-983da380-fcb4-11e9-86e9-eae6026a7e8f.png)
![image](https://user-images.githubusercontent.com/35050056/68006864-e357b680-fcb4-11e9-96e4-f0a72deb0d3e.png)


Seems like you need the following attribtues:
```
role="switch"
aria-checked="true" | "false"
```

The `data` attributes are unnecessary.